.github/workflows/build-deploy-dev.yml

```diff
@@ -197,23 +196,27 @@ jobs:
       - name: Log in to OpenShift
         run: |
           oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
-      # Deploy SIMS-Api
-      - name: Deploy SIMS-API
-        working-directory: "./devops/"
-        run: |
-          make oc-deploy-api
-      # Deploy Workers
-      - name: Deploy Workers
+          if [ $? -ne 0 ]; then
```

Comment:
When the steps failed they usually will indicate it in a nice way in the UI, why do we need this additional check here?
For instance,
![image](https://github.com/bcgov/SIMS/assets/61259237/30e474cf-cd12-4686-bafb-996db29ce1d4)

Reply:
You are correct that failed steps in a GitHub Actions workflow typically indicate the failure in the GitHub UI, making it easy to identify the cause of the failure. The additional check in the provided code snippet is a manual validation step that provides an explicit error message and exits the workflow with a failure status code.
Also by including the check and exit statement, the workflow consistently handles OpenShift login failures. It ensures that the workflow stops and reports a failure in a predictable manner, regardless of any potential variations in the behavior or error reporting of the oc login command itself.

Reply:
Thanks for reverting these changes as per the Team discussion.